### ISSUE TITLE
Generate uftrace.data in rotation

### ISSUE THREAD

**Issue by MichelleJin12** [namhyung/uftrace#1920]
If I trace long enough, uftrace.data will become too large to view with Chrome Trace Viewer, tui, replay, etc.
What about splitting up uftrace.data based on file size or time?

reference: https://linux.die.net/man/8/logrotate

**Comment by namhyung** [comment]
Yeah, I think @honggyukim raised the same issue already.  We need to think about how to make it smooth. :)
